versions/3.1.1.md

```diff
@@ -182,12 +182,48 @@ It is the responsibility of an embedding format to define how to parse embedded
 
+#### <a name="resolvingIndirectLinkages"></a>Resolving Indirect Connections
+
+Several features of this specification require resolving a non-URI-based connection to some other part of the OpenAPI Description (OAD).
+
+These connections are easily resolved in single-document OADs, but the resolution process in multi-document OADs has never been spelled out, and is therefore _implementation-defined_, within the constraints described in this section.
+In some cases, an unambiguous URI-based alternative is available, and OAD authors are RECOMMENDED to always use the alternative:
+
+Source | Target | Alternative
+------ | ------ | -----------
+[Security Requirement Object](#securityRequirementObject) `{name}` |  [Security Scheme Object](#securitySchemeObject) name under the [Components Object](#componentsObject) | _n/a_
+[Discriminator Object](#discriminatorObject) `mapping` _(implicit, or explicit name syntax)_ | [Schema Object](#schemaObject) name under the Components Object | `mapping` _(explicit URI syntax)_
+[Link Object](#linkObject) `operationId` | [Path Item Object](#pathItemObject) `operationId` | `operationRef`
+
+A fourth implicit connection, which involves appending the templated URL paths of the [Paths Object](#pathsObject) to the appropriate [Server Object](#serverObject)'s `url` field, is unambiguous because only the entry document's Paths Object contributes URLs to the described API.
+
+For resolving indirect connections from a referenced (non-entry) document, it is RECOMMENDED that tools support one or both of the following approaches, and it is likewise RECOMMENDED that OAD authors organize their documents to work well with them.
+
+##### OADs Using Only Complete Documents
+
+The approach in this section is RECOMMENDED for OADs that are composed entirely of complete OpenAPI documents, optionally including standalone JSON Schema documents as long as any Discriminator Objects within them only use explicit `mapping` URIs for all possible values.
+This OAD design is RECOMMENDED when sharing components among multiple OADs.
+
+In this approach, each document is self-contained and resolves components to its own Components Object.
+When resolving an `operationId`, all Path Item Objects from all parsed documents are considered.
+This requires ensuring that all referenced documents have been parsed prior to determining an `operationId` to be unresolvable.
+
+**Note:** Currently, Security Requirement Objects do not support a URI-based mechanism for connecting to Security Scheme Objects, which means that in this approach, they can only connect to Security Schemes within the same document.
+This limitation is expected to be addressed in the next minor version of this specification.
```

Comment:
```suggestion
This limitation is expected to be addressed in a future version of this specification.
```
Don't over-promise.

Reply:
The PR for this is up for 3.2, so if we can agree on it, then "next minor" is correct and sets a better expectation.

Reply:
On second thought, it's better to be more vague as we can always add specifics later if we agree to the 3.2 change.  But I don't want us to have to think about 3.2 before doing these patch releases, so I'll commit this suggestion.